src/Components/MoodRating/MoodRating.js

```diff
@@ -0,0 +1,150 @@
+import React, { useState, useContext } from 'react'
+import style from './MoodRating.module.scss'
+import { ViewContext } from '../../Context/ViewContext'
+import { SessionContext } from '../../Context/SessionContext'
+import { postSession } from '../../apiCalls'
+import moodIcon1 from '../../assets/moodIcons/moodIcon1.png'
+import moodIcon2 from '../../assets/moodIcons/moodIcon2.png'
+import moodIcon3 from '../../assets/moodIcons/moodIcon3.png'
+import moodIcon4 from '../../assets/moodIcons/moodIcon4.png'
+import moodIcon5 from '../../assets/moodIcons/moodIcon5.png'
+
+const MoodRating = () => {
+  const [ moodRating, setMoodRating ] = useState(null)
+  const [ error, setError ] = useState(false)
+  const [ view, setView ] = useContext(ViewContext)
+  const [ session, setSession ] = useContext(SessionContext)
+
+  const rateMood = (event) => {
+    setMoodRating(event.target.name)
+    if (view === 'mood-rating-1') {
+      setSession({
+        ...session,
+        moodRating1: event.target.name
+      })
+    } else if (view === 'mood-rating-2') {
+      setSession({
+        ...session,
+        moodRating2: event.target.name
+      })
+    }
+    setError(false)
+  }
+
+  const recordMood = () => {
+    if (view === 'mood-rating-1') {
+      setView('content-selection')
+    } else if (view === 'mood-rating-2') {
+      setView('timer')
+      postSession(session)
+    }
+  }
+
+  const checkSubmit = (event) => {
+    event.preventDefault()
+    if(!moodRating) {
+      setError(true)
+    } else {
+      recordMood()
+    }
+  }
+
+  return (
+    <section className={style.moodRatingContainer}>
```

Comment:
I love how this is looking - really effective, eye-pleasing, and in line with the Som Timer™ brand

Reply:
Absolutely, I'm really excited about this feature.